src/react-apps/applications/runtime/src/features/instantiate/containers/PartySelection.tsx

```diff
@@ -182,12 +181,12 @@ function PartySelection(props: IPartySelectionProps) {
     if (!language || !language.party_selection) {
       return null;
     }
-    if (location.state !== undefined &&
-      (location.state as IRedirectReason) &&
-      (location.state as IRedirectReason) !== undefined) {
-        switch ((location.state as IRedirectReason).errorType) {
+    if ((match.params as IRedirectParams).errorCode !== null) {
+      try {
+        const errorCode: number = parseInt((match.params as IRedirectParams).errorCode.toString(), 10);
```

Comment:
Her kan du velge selv, jeg stusset på hva 10 gjorde der men sjekket parseInt og det er radix aka base10 som du sier her.
Forslaget mitt er å enten la den være eller ha en variabel / konstant som heter Base10 ellr no.
Men helt opp til deg :)

Reply:
I think having a const is just unnecessary since I only added the 10 because ts-lint was complaining about adding radix. This is the only place I know of that we use parseInt.